procedures/unit-testing-autorun.ipf

```diff
@@ -109,7 +110,13 @@ Function/S GetBaseFilename()
 	if(strlen(baseFilename))
 		return baseFilename
 	endif
-	sprintf baseFilename, "%s_%s_%s", IgorInfo(1), Secs2Date(DateTime, -2), ReplaceString(":", Secs2Time(DateTime, 3), "-")
+
+	if(SVAR_Exists(baseFilenameOverwrite) && strlen(baseFilenameOverwrite))
```

Comment:
I know its intriguing to just copy the strlen check from above, though we use now UTF_Utils#IsEmpty(str) for this.
The advantage of UTF_Utils#IsEmpty is that it also checks for a null string, where strlen would cause an RTE if you can resolve the baseFileNameOverwrite global and baseFileNameOverwrite is a null string.

Reply:
I created #303 to target the `strlen()` issue in general. The requested place will be fixed of course.

Reply:
For documentation: I have to revert to `strlen` at this place. `baseFilenameOverwrite` is an `SVAR` and therefore cannot be null and `UTF_Utils#IsEmpty` is only accepting strings.